<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="general">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>21</y>
       <width>331</width>
       <height>43</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lang_label">
        <property name="text">
         <string>Language</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="language">
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>0</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Application restart is required</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="enable_filter_blocked">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable filter blocked contacts</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="privacy">
    <attribute name="title">
     <string>Privacy</string>
    </attribute>
    <widget class="QLabel" name="chat_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Allow chat message from...</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="chat_contacts">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>People from my Contacts</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">allow_chat</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="chat_authorized">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Authorized Contacts</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">allow_chat</string>
     </attribute>
    </widget>
    <widget class="QLabel" name="mail_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Allow mail message from...</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="mail_anyone">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>191</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Anyone</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">allow_mail</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="mail_authorized">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Authorized Contacts</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">allow_mail</string>
     </attribute>
    </widget>
    <widget class="QCheckBox" name="save_spam">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Blocked message save in Spam folder</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="bitshares_wallet">
    <attribute name="title">
     <string>BitShares Wallet</string>
    </attribute>
    <widget class="QCheckBox" name="wallets_client_on_startup">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Start wallets client on startup Keyhotee</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mail_authorized</sender>
   <signal>toggled(bool)</signal>
   <receiver>save_spam</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="allow_mail"/>
  <buttongroup name="allow_chat"/>
 </buttongroups>
</ui>
